tools/depends/target/boost/Makefile

```diff
@@ -15,23 +15,15 @@ $(TARBALLS_LOCATION)/$(ARCHIVE):
 $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM); mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
-	echo 'using gcc : : $(CC) : ' > $(PLATFORM)/tools/build/v2/user-config.jam
-	echo '	<cflags>"$(CFLAGS)"' >> $(PLATFORM)/tools/build/v2/user-config.jam
-	echo '  <cxxflags>"$(CXXFLAGS)"' >> $(PLATFORM)/tools/build/v2/user-config.jam
-	echo '	;' >> $(PLATFORM)/tools/build/v2/user-config.jam
-	cd $(PLATFORM); patch -p1 < ../add-arm-mem-barrier.patch
-	cd $(PLATFORM); patch -p1 < ../fix-deprecated-swp.patch
 ifeq (arm, $(findstring arm, $(CPU)))
-	cd $(PLATFORM); patch -p0 < ../fix_shared_ptr_multithread.patch
+	#ugly hack to avoid issue with line endings. boost headers use \r\n and patch
+	#will be converted by git to \n and fail to apply
+	cd $(PLATFORM); echo "#define BOOST_SP_USE_SPINLOCK" >> boost/config/user.hpp
 endif
-	cd $(PLATFORM); patch -p1 < ../fix-self-assignment-warnings.patch
-	cd $(PLATFORM); ./bootstrap.sh --prefix=$(PREFIX)
+	cd $(PLATFORM); cp -r boost $(PREFIX)/include/
```

Comment:
Why a plain "cp"?
bootstrap/bjam is still the recommended install way: http://theboostcpplibraries.com/introduction-installation